### ISSUE TITLE
One-line constructors with braces in the body are marked as violations

### ISSUE THREAD

**Issue by Brickster** [CodeNarc/CodeNarc#264]
This can be reproduced by adding the following test to `SpaceAfterOpeningBraceRuleTest`.

```
@Test
void testApplyTo_OneLineConstructorBodyContainsBrace_NoViolations() {
    final SOURCE = '''
        class MyClass {
            String s
            MyClass() { s = '{"json": true}' }
        }
    '''
    assertNoViolations(SOURCE)
}
```

**Comment by anjaliraman** [comment]
So does this mean to suggest that when something is between single quotes it should not check for spaces?

**Comment by Brickster** [comment]
Ideally, anything that isn't the open brace for the code block would be ignored. So open brace in strings, comments, etc.

**Comment by chrismair** [comment]
Fix will be available in CodeNarc 1.1
